public/app/plugins/datasource/elasticsearch/plugin.json

```diff
@@ -3,27 +3,29 @@
   "name": "Elasticsearch",
   "id": "elasticsearch",
   "category": "logging",
-
   "info": {
     "description": "Open source logging & analytics database",
     "author": {
       "name": "Grafana Labs",
       "url": "https://grafana.com"
     },
-    "keywords": ["elasticsearch"],
+    "keywords": ["elasticsearch", "datasource", "database"],
```

Comment:
should we also add the "logs" or "metrics" keywords here?

Reply:
good point, added in latest commit 👍